libs/ui/layout/src/lib/cards-subsections/cards-subsections.component.ts

```diff
@@ -0,0 +1,138 @@
+import {
+  AfterViewInit,
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ContentChildren,
+  ElementRef,
+  Input,
+  QueryList,
+} from '@angular/core'
+
+import { CommonModule } from '@angular/common'
+import { Paginable } from '../paginable.interface'
+import { PaginationDotsComponent } from '../pagination-dots/pagination-dots.component'
+import { TranslateModule } from '@ngx-translate/core'
+import { PreviousNextButtonsComponent } from '../previous-next-buttons/previous-next-buttons.component'
+
+enum ComponentSize {
+  LARGE = 'L',
+  MEDIUM = 'M',
+  SMALL = 'S',
+  EXTRA_SMALL = 'XS',
+}
+
+@Component({
+  selector: 'gn-ui-cards-subsections',
+  templateUrl: './cards-subsections.component.html',
+  styleUrls: ['./cards-subsections.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  standalone: true,
+  imports: [
+    CommonModule,
+    PaginationDotsComponent,
+    TranslateModule,
+    PreviousNextButtonsComponent,
+  ],
+})
+export class CardsSubsectionsComponent implements AfterViewInit, Paginable {
+  @ContentChildren('subComponent', { read: ElementRef })
+  subComponents: QueryList<ElementRef<HTMLElement>>
+  @Input() title: string
+
+  get paginableElement(): Paginable {
+    return this
+  }
+
+  protected minHeight = 0
+  protected subComponentSize: ComponentSize = ComponentSize.MEDIUM
```

Comment:
I think using string literals here instead of an enum is better, so:

`protected subComponentSize: 'M' | 'S' | 'XS' = 'M'`